src/main/java/fr/greencodeinitiative/python/PythonRuleRepository.java

```diff
@@ -44,9 +44,9 @@ public class PythonRuleRepository implements RulesDefinition, PythonCustomRuleRe
     );
 
     public static final String LANGUAGE = "py";
-    public static final String NAME = "ecoCode";
-    public static final String RESOURCE_BASE_PATH = "io/ecocode/rules/python";
-    public static final String REPOSITORY_KEY = "ecocode-python";
+    public static final String NAME = "creedengo";
+    public static final String RESOURCE_BASE_PATH = "io/creedengo/rules/python";
```

Comment:
this renaming task on RESOURCE_BASE_PATH has to be done in another PR once "ecocode" repository will be renamed